config/features.yml

```diff
@@ -171,6 +171,10 @@ features:
     actor_type: user
     description: Diverts codepath to LocalBGSRefactored
     enable_in_development: true
+  claims_api_hardcode_wsdl:
```

Comment:
should we `lighthouse_` this ... there's quite a few "claims" namespaces in vets-api currently. May provide a bit more clarity it's ours and not someone else's.